scenarios/ocd/CreateContainerAppDeploymentFromSource/README.md

```diff
@@ -7,21 +7,23 @@ In this guide, we'll be walking through deploying the necessary resources for a
 - **Azure Computer Vision** to analyze the images for cats or dogs
 - **Azure Container App** to deploy our code
 
+Note: If you've never created a Computer Vision resource before, you will not be able to create one using the Azure CLI. You must create your first Computer Vision resource from the Azure portal to review and acknowledge the Responsible AI terms and conditions. You can do so here: [Create a Computer Vision Resource](https://portal.azure.com/#create/Microsoft.CognitiveServicesComputerVision). After that, you can create subsequent resources using any deployment tool (SDK, CLI, or ARM template, etc) under the same Azure subscription.
+
 ## Define Environment Variables
 
-The first step in this tutorial is to define environment variables. Replace the values with your own.
+The first step in this tutorial is to define environment variables. **Replace the values on the right with your own unique values.** These values will be used throughout the tutorial to create resources and configure the application. Use lowercase and no special characters for the storage account name.
 
 ```plaintext
-export MY_RESOURCE_GROUP_NAME=myresourcegroup
```

Comment:
These values should be set back to their original values, but you should add a unique postfix to the end of each of them so that each run of the document generates a new set of resources.